src/plugins/saved_objects_management/public/utils.ts

```diff
@@ -25,9 +25,10 @@ export function formatInspectUrl(
   const useUpdatedUX = !!coreStart.uiSettings.get('home:useNewHomePage');
   let finalEditUrl = editUrl;
   if (useUpdatedUX && finalEditUrl) {
-    finalEditUrl = finalEditUrl.replace(/^\/management\/opensearch-dashboards/, '/app');
+    finalEditUrl = finalEditUrl.replace(/^\/management\/opensearch-dashboards/, '');
   }
   if (finalEditUrl) {
+    finalEditUrl = `/app${finalEditUrl}`;
```

Comment:
is there a reason to do this?

Reply:
In any case, need to add the `\app` prefix to the url, otherwise user will not be able to access it. According to previous practice, when new home is closed, the URL will miss the `\app` prefix